apps/fabric-website/src/pages/Controls/TextPage/docs/android/TextImplementation.md

```diff
@@ -0,0 +1,27 @@
+### Style names
+
+`TextAppearance.UIFabric.Headline`
+
+`TextAppearance.UIFabric.Title1`
+
+`TextAppearance.UIFabric.Title2`
+
+`TextAppearance.UIFabric.Heading`
+
+`TextAppearance.UIFabric.SubHeading`
+
+`TextAppearance.UIFabric.SubHeading2`
+
+`TextAppearance.UIFabric.Body1`
+
+`TextAppearance.UIFabric.Body2`
+
+`TextAppearance.UIFabric.Caption`
+
+### Library
+
+[UI Fabric Android](https://github.com/OfficeDev/ui-fabric-android)
+
+[Sample Code in activity](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/java/com/microsoft/officeuifabricdemo/demos/TypographyActivity.kt)
```

Comment:
Thinking these should either be completely Title Case or completely Sentence case, rather than switching in the middle?